articles/quickstart/backend/java-spring-security5/01-authorization.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Authorization
+name: How to secure your API using Spring Security 5 and Auth0
+description: This tutorial demonstrates how to add authorization to a an API using Spring Security 5.
+budicon: 500
+topics:
+    - quickstart
+    - backend
+    - java
+    - spring-security
+github:
+    path: 01-Authorization-MVC
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../../../_includes/_api_auth_intro') %>
+
+<%= include('../_includes/_api_create_new') %>
+
+<%= include('../_includes/_api_auth_preamble') %>
+
+## Configure the Sample Project
+
+::: note
+This Quickstart uses Spring MVC. If you are using Spring WebFlux, the steps to secure an API are similar, but some of the implementation details are different. Refer to the [Spring Security WebFlux Sample Code](https://github.com/auth0-samples/auth0-spring-security5-api-sample/tree/master/01-Authorization-WebFlux) to see how integrate Auth0 with your Spring WebFlux API.
+:::
+
+The sample project uses a `/src/main/resources/application.yml` file, which configures it to use the correct Auth0 **Domain** and **API Identifier** for your API. If you download the code from this page it will be automatically configured. If you use the example from GitHub, you will need to fill it in yourself.
+
+```yaml
+auth0:
+  audience: ${apiIdentifier}
+spring:
+  security:
+    oauth2:
+        resourceserver:
+            jwt:
+                issuer-uri: https://${account.namespace}/
+```
+
+| Attribute | Description|
+| --- | --- |
+| `auth0.audience` | The unique identifier for your API. If you are following the steps in this tutorial it would be `https://quickstarts/api`. |
```

Comment:
if you already know the value (you're mentioning it here), why not use that instead in the snippet above?

Reply:
In the snippet above we use the `apiIdentifier` value which will be replaced with the value for the chosen API in the dropdown widget in the code snippet viewer. So it shouldn't be hard-coded in that case, so here it makes sense to call out if following along manually.

Here's a screenshot of the docs rendering the code snippet above:

![image](https://user-images.githubusercontent.com/276225/61669560-1649e180-aca6-11e9-8f3d-03fb31cd99a6.png)